practices/responsibility.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: responsibility
+stages:
+  - development
+short_description: Responsibility is about trusting people and their commitment towards the work assigned. A responsible person completes the task assigned to them no matter what the circumstances are.
+tags:
+  - responsibility
+  - team work
+  - ability
+  - response
+  - appreciation
+  - problems
+  - problem-solving
+  - assign
+  - leader
+
+keywords:
+  - responsibility
+  - trust
+  - people
+  - benefits
+  - team lead
+---
+
+# Responsibility
+
+**TL;DR**
+Responsibility is about trusting people and their commitment towards the work assigned. A responsible person completes the task assigned to them no matter what the circumstances are.
+
+## What Is Responsibility
+
+**Response + ability**, these two words combine to form a well known word **Responsibility**. It means the ability of a person to respond to a situation according to its capabilities. In tough situations, a responsible person will never shy away from its responsibilities, instead it will rise to the occasion to save the sinking ship. Taking responsibility requires a great sense of courage as well because if the person fails he or she not only fails themselves but also the entire team that are dependent on them. However, taking responsibility beyond oneself means a greater level of risk and therefore a greater degree of success.
+
+Responsibility is not for everyone but who chooses to become responsible for their actions, achieves a greater sense of power and freedom.
+
+[Responsibility](/files/responsibility.jpg)
+![Responsibility](https://shiftyourfamilybusiness.com/wp-content/uploads/2018/05/ResponsibilityBlog-845x321.jpg)
+
+## Why You Might Want Responsibility
+
+Responsibility brings us many folds of its benefits. It helps an individual achieve its goals and complete the assigned work before deadline. It helps the company or oganization work more efficiently and keep its client happy.
+
+From a team point of view, if we have more responsible people in our team, the better will be our performance: 
+
+- Each team member will be able to work more independently and make better decisions. 
+- We can trust the team members about the work assigned to them.
+- The team members will also tend to be more honest and keep their promises.
+- Responsibility is a part of healthy [team culture](https://preview.developerexperience.io/practices/team-culture)
+
+## Problems Responsibility Solves
+
+- Creates a positive atmosphere.
+- Matches the expectations of the company.
+- Increases the self worth of an individual, as the work done by them is appreciated.
+- Quality of work is higher and no poor quality work is done | [Poor Code Quality](https://developerexperience.io/problems/poor-code-quality)
+- Team focuses mostly on solving problems rather than indulging in any unhealtly blame game | [Demotivated Team](https://developerexperience.io/problems/demotivated-team)
```

Comment:
```suggestion
- Team focuses mostly on solving problems rather than indulging in any unhealthy blame game | [Demotivated Team](/problems/demotivated-team)
```